src/IteratorToArrayRecursiveCapable.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Util\Normalization\NormalizeIterableCapableTrait;
+use InvalidArgumentException;
+use stdClass;
+use Traversable;
+
+/**
+ * Trait IteratorToArrayConvertCapable, adds ability to convert an iterator to an array.
+ *
+ * @since [*next-version*]
+ */
+trait IteratorToArrayRecursiveCapable
```

Comment:
Much better! But you forgot the `Trait` suffix! :stuck_out_tongue:

Reply:
Shame on me! Fixing.